frontend/src/views/WasteActionsPage/WasteActionPage.vue

```diff
@@ -31,8 +31,20 @@
           <p class="mt-12">{{ wasteAction.subtitle }}</p>
           <p v-html="wasteAction.description" class="mt-9"></p>
         </v-col>
-        <v-col cols="12" v-bind:class="{ 'mt-7': $vuetify.breakpoint.smAndUp }" sm="2">
+        <v-col v-if="loggedUser" cols="12" class="d-flex flex-column align-start mt-8" sm="2">
           <!-- Implement action buttons -->
+          <p class="mb-2">Mis en place</p>
+          <DsfrTagGroup
+            v-if="hasActionsDone"
```

Comment:
```suggestion
            v-if="actionsDone && actionsDone.length"
```
suggestion pour raccourcir le code en bas